proprietary/tokens/src/brand/amsterdam/space.tokens.json

```diff
@@ -8,7 +8,14 @@
         "comment": "Grows from 16px at 320px wide to 56px at 1600px wide."
       },
       "lg": { "value": "clamp(1.5rem, calc(4.6875vw + 0.5625rem), 5.25rem)" },
-      "xl": { "value": "clamp(2rem, calc(6.25vw + 0.75rem), 7rem)" }
+      "xl": { "value": "clamp(2rem, calc(6.25vw + 0.75rem), 7rem)" },
+      "inside": {
```

Comment:
After reading the docs I would expect a different structure here:

* space
	* grid
		* xs
		* sm
		* md
		* lg
		* xl
	* stack
		* xs
		* sm
		* md
		* lg
		* xl
	* inside
		* xs
		* sm
		* md
		* lg
		* xl
	* type
		* xs
		* sm
		* md
		* lg
		* xl

Reply:
Yes, that’s how it will be. I mentioned having to group the grid tokens in the PR description.

Reply:
I’ve prepared that separate PR: #1098